webform_prefill.js

```diff
@@ -7,17 +7,13 @@
 
 Drupal.behaviors.webform_prefill = {
   attach: function (context, settings) {
-
-    if (!$.isFunction($.fn.formPrefill)) {
-      return;
-    }
-
-    var settings = Drupal.settings.webform_prefill || {
+    var settings = settings.webform_prefill || {
       map: {},
       storage: ['sessionStorage'],
       cookieDomain: ''
     };
 
+    // This also triggers reading values from the URL even if there is no form.
```

Comment:
How does that work when `$('form.webform-client-form', context)` is empty?

Reply:
`$.fn.formPrefill` is still called on the empty result. It does most of its work *before* attempting to loop over the empty result set.

Reply:
I'm pretty sure this is against some best practice :rofl:

From the [jQuery learning center](https://learn.jquery.com/plugins/):
"The idea of a plugin is to do something with a collection of elements."